### PR TITLE
[WJ-243] Add more documentation about development processes

### PR DESCRIPTION
**Pull request by emmiegit**
This issue has been outstanding for a bit, and is marked as necessary for `1.0.0`. I added a bit more relevant information about our development processes to be helpful to interested potential contributors.

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+This document assumes you have read [Development.md](development.md).
+
+Wikijump has a fairly extensive scope and has lots of areas for people to contribute. Whether it's a small typofix or work on a potential new feature, there's always room for community contributions.
+
+It is important that you join the Wikijump discord guild so you can discuss and coordinate with the Wikijump team.  If appropriate, a team member can make a [JIRA](https://scuttle.atlassian.net/browse/WJ) issue for it. You can get an invitation by asking in [#site11](https://scp-wiki.wikidot.com/chat-guide).
```
Comment (rossjrw) [inline docs/contributing.md]:
```suggestion
Wikijump has a fairly extensive scope and has lots of areas for people to contribute. Whether it's a small typofix or work on a potential new feature, there's always room for community contributions. If appropriate, a team member can make a [JIRA](https://scuttle.atlassian.net/browse/WJ) issue for it.

It is important that you join the Wikijump discord guild so you can discuss and coordinate with the Wikijump team.  You can get an invitation by asking in [#site11](https://scp-wiki.wikidot.com/chat-guide).
```

Comment (rossjrw) [inline docs/contributing.md]:
```suggestion
Wikijump has a fairly extensive scope and has lots of areas for people to contribute. Whether it's a small typofix or work on a potential new feature, there's always room for community contributions.

It is important that you join the Wikijump Discord so you can discuss and coordinate with the Wikijump team.  If appropriate, a team member can make a [JIRA](https://scuttle.atlassian.net/browse/WJ) issue for it. You can get an invitation by asking in [#site11](https://scp-wiki.wikidot.com/chat-guide).
```

**Review by rossjrw — APPROVED** [review]

File: docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+This document assumes you have read [Development.md](development.md).
+
+Wikijump has a fairly extensive scope and has lots of areas for people to contribute. Whether it's a small typofix or work on a potential new feature, there's always room for community contributions. If appropriate, a team member can make a [JIRA](https://scuttle.atlassian.net/browse/WJ) issue for it.
+
+It is important that you join the Wikijump discord guild so you can discuss and coordinate with the Wikijump team.  You can get an invitation by asking in [#site11](https://scp-wiki.wikidot.com/chat-guide).
```
Comment (emmiegit) [inline docs/contributing.md]:
```suggestion
It is important that you join the Wikijump Discord so you can discuss and coordinate with the Wikijump team.  You can get an invitation by asking in [#site11](https://scp-wiki.wikidot.com/chat-guide).
```

**Comment by emmiegit** [comment]
thanks @rossjrw

**Pull request merged**